### PR TITLE
Mark notification as disabled only if authorization status is denied (Fix #219)

### LINKED ISSUE #219: Bug: The preferences has a “⚠️ Your macOS notification settings..” even though notifications for MeetingBar are enabled.
*from email*

Maybe it's because we only check for an Authorized state without taking Provisional or Ephemeral into account. 
https://github.com/leits/MeetingBar/blob/fd58e69906447d6724b3edeef7eb26c63a7814c7/MeetingBar/Views/Shared.swift#L27

### PR DESCRIPTION
**Pull request by leits**
### Status
**READY**

### CODE REVIEW

**Comment by leits** [comment]
Fix for #219

**Comment by leits** [comment]
@jgoldhammer, do you have any concerns?

**Review by jgoldhammer — APPROVED** [review]

File: MeetingBar/Views/Shared.swift [hunk 1]
```diff
@@ -24,7 +24,7 @@ struct JoinEventNotificationPicker: View {
 
         center.getNotificationSettings { notificationSettings in
             noAlertStyle = notificationSettings.alertStyle != UNAlertStyle.alert
-            notificationsDisabled = notificationSettings.authorizationStatus != UNAuthorizationStatus.authorized
+            notificationsDisabled = notificationSettings.authorizationStatus != UNAuthorizationStatus.denied
```
Comment (jgoldhammer) [inline MeetingBar/Views/Shared.swift:27]: Is this correct? I think it must be == instead of !=

Comment (leits) [reply MeetingBar/Views/Shared.swift:27]: You're absolutely right! Thanks!

Comment (leits) [reply MeetingBar/Views/Shared.swift:27]: You're absolutely right!

**Pull request merged**